Enterprise-CMCS/macpro-mdct-mfp
# Setup api for create fetch

### Description


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/241785f3-c94a-4e65-bac5-caffcc58f748

### Related ticket(s)


MDCT-2723

---
### How to test


- ./dev local
- Go to /standard
- Open up the console/network tab and take a peak!
- You can also use DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin and see the form there

### Important updates




---
### Author checklist
